wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -329,7 +329,7 @@ The third object is a slider:
 - At the position (100,100)
 - At 12.6 seconds
 - Starting a new combo
-- With a compound bézier curve slider body, where the first curve's control points are (100,100), (200,200), and (250,200), and the second curve's control points are (250,200), (300,150)
+- With a compound bézier curve slider body, where the curve's control points are (100,100), (200,200), (250,200), (250,200), and (300,150)
```

Comment:
I think I got wrong between control point and anchor point
The original description seems no problem
so I do some change with description of `curvePoints` (Line 260)

Reply:
Control point is the terminology used for beziér curves. Anchor point is what osu! calls them. In practice they mean the same thing, since two white anchor points form a red anchor point and similarly, two control points on top of each other denote a separation of beziér curves in a compound curve.